charts/app-config/templates/_helpers.tpl

```diff
@@ -84,3 +84,24 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Service name to be used by Sentry
+*/}}
+{{- define "app-config.sentryName" -}}
+{{- if .Values.sentry.service.name }}
+{{- printf "%s" .Values.sentry.service.name -}}
```

Comment:
Could you please explain why these should be configurable?

Reply:
Those two don't need to be configurable, I made it that way just to give more flexibility in how the service names are displayed in Sentry as an extra option. I can remove that, if this is problematic or just not needed.

Reply:
It's not problematic and in general it's good to have the flexibility. However this adds a bit of complexity to the charts and since I haven't used the Sentry interface I don't know how much benefit this brings. You can decide on this.

Reply:
Ok, thanks for clarification, I removed the conditional values for now.

Reply:
Thanks, I think in this case we can use existing chart variables such as `{{ include "catalog-server.name" . }}`, `{{ include "catalog-server.chart" . }}`, `{{ .Release.Name }}`, `{{ .Release.Service }}` instead of creating new functions in `_helpers.tpl`.

Reply:
@pvannierop, before I change something here, could you double-check if this is what you had in mind for those 2 variables (sentryName, sentryRelease), or you think it is better to use a different combination of variables for sentryName, e.g. instead of the fullName defined above, fix it to some string containing .Chart.Name .Chart.Version or .Release.Name